packages/cfd/src/Containers/trade-modal.tsx

```diff
@@ -71,31 +71,55 @@ const TradeModal = ({
     const downloadCenterAppOption = (platform_type: TCFDsPlatformType) => {
         if (platform_type === 'dxtrade') {
             return (
-                <div className='cfd-trade-modal__download-center-app--option'>
-                    <Text className='cfd-trade-modal__download-center-app--option-item' size='xs'>
-                        {localize('Run Deriv X on your browser')}
+                <>
+                    <div className='cfd-trade-modal__download-center-app--option'>
```

Comment:
You are repeating the same code. You can use variables in your Localize to show different texts depending on your variable